### PR TITLE
Fixes #31519 - extra JSON kexec argument fixed

### PR DESCRIPTION
**Pull request by lzap**
It was appended as array (into array) which is causing problems when executing the command later on. This is a regression which slipped in, in my testing I did not use any "extra" arguments. I tested this, just read the code and merge for me thanks.

### CODE REVIEW

File: lib/smart_proxy_discovery_image/power_api.rb [hunk 1]
```diff
@@ -27,7 +27,12 @@ class PowerApi < ::Sinatra::Base
         log_halt 500, "Unable to parse kexec JSON input: #{body_data}"
       end
       args = ["--debug", "--force"]
-      args << data['extra'] if data['extra']
+      extra = data['extra']
+      if extra && extra.is_a?(String)
```
Comment (adamruzicka) [inline lib/smart_proxy_discovery_image/power_api.rb:31]: You can safely call `is_a?(klass)` on `nil` so the leading `extra && ` is not really necessary

**Review by adamruzicka — COMMENTED** [review]
One nitpick, otherwise LGTM

File: lib/smart_proxy_discovery_image/power_api.rb (hunk 1 above)
Comment (lzap) [reply lib/smart_proxy_discovery_image/power_api.rb:31]: Thanks, noted. I am gonna merge this as is since I tested it tho :-)

**Comment by lzap** [comment]
1.3.2 pushed out https://github.com/theforeman/foreman-packaging/pull/6148

**Pull request merged**